openmc/tallies.py

```diff
@@ -1729,10 +1749,14 @@ def swap_filters(self, filter1, filter2):
         filter2 : Filter
             The filter to swap with filter1
 
+        inline : bool, optional
```

Comment:
I see that this is the case for several other routines in "tallies.py", looks like I screwed this up in a previous commit.